benchmark/synthetic_benchmark.py

```diff
@@ -20,10 +20,8 @@
     formatter_class=argparse.ArgumentDefaultsHelpFormatter,
 )
 
-parser.add_argument("--model", type=str, default="resnet50",
-                    help="model to benchmark")
-parser.add_argument("--batch-size", type=int,
-                    default=32, help="input batch size")
+parser.add_argument("--model", type=str, default="resnet50", help="model to benchmark")
```

Comment:
not related to this pr, make it another one

Reply:
Just black format the code. It is necessary and hard to recover..